### ISSUE TITLE
aXe flags use of ARIA 1.1 aria-current attribute as invalid.

### ISSUE THREAD

**Issue by Wildebrew** [dequelabs/axe-core#270]
The ARIA 1.1 spec has a few new attributes. One of them is [aria-current](http://www.w3.org/TR/wai-aria-1.1/#aria-current).
aria-current is used to programmatically mark the currently emphasized or selected element in a set of elements.
aXe fails use of aria-current="true" under "ARIA attributes must conform to valid names" rule (see attached example).
A.T. support for ARIA 1.1 attributes is still largely missing but the markup is valid and an ARIA parser hould consider the latest spec update.
I suggest that aria-current be added and its valid use be passed by aXe (see link to spec re valid values).
If not, at least only flag use of aria-current as a warning due to incompatibility with assistive technology (this attribute is supported in Jaws 18 with IE/Firefox, but not yet in other browser/A.T. combinations).

**Comment by Wildebrew** [comment]
[aria-current.txt](https://github.com/dequelabs/axe-core/files/664358/aria-current.txt)

**Comment by marcysutton** [comment]
We've added support for this already internally, but it won't be available to the public until the next open source release.

**Comment by marcysutton** [comment]
Reopening this issue until we do a merge from our internal fork.

**Comment by marcysutton** [comment]
This should be merged to master now. Stay tuned for the axe-core 2.2.0 release.
